app/handler/dmarc.py

```diff
@@ -34,22 +34,46 @@ def apply_dmarc_policy_for_forward_phase(
 
     from_header = get_header_unicode(msg[headers.FROM])
 
+    warning_plain_text = f"""This email failed anti-phishing checks when it was received by SimpleLogin, be careful with its content.
+More info on https://simplelogin.io/docs/getting-started/anti-phishing/
+            """
+    warning_html = f"""
+        <p style="color:red">
+            This email failed anti-phishing checks when it was received by SimpleLogin, be careful with its content.
+            More info on <a href="https://simplelogin.io/docs/getting-started/anti-phishing/">anti-phishing measure</a>
+        </p>
+        """
+
+    # do not quarantine an email if fails DMARC but has a small rspamd score
+    if (
+        config.MIN_RSPAMD_SCORE_FOR_FAILED_DMARC is not None
+        and spam_result.rspamd_score < config.MIN_RSPAMD_SCORE_FOR_FAILED_DMARC
```

Comment:
```suggestion
        spam_result.rspamd_score < config.MIN_RSPAMD_SCORE_FOR_FAILED_DMARC
```

Reply:
I want to make sure ` config.MIN_RSPAMD_SCORE_FOR_FAILED_DMARC` is set and not use the implicit default value.

Reply:
I prefer this to be explicit actually.